### PR TITLE
Source branch for GitHub Pages has to be set

### PR DESCRIPTION
**Pull request by SchulteMarkus**
Using your awesome workflow, a branch gh-pages is being created, which serves as GitHub Pages source. This has to be set once in the repository settings.

### CODE REVIEW

File: docs/publishing-your-site.md [hunk 1]
```diff
@@ -91,6 +91,9 @@ Now, when a new commit is pushed to either the `master` or `main` branches,
 the static site is automatically built and deployed. Push your changes to see
 the workflow in action.
 
+After the first successful build, visit `github.com/<username>/<repository>/settings/pages`
```
Comment (squidfunk) [inline docs/publishing-your-site.md:94]: Is this really necessary? When I tested it, it always automatically set the default branch, unless you changed it.

Comment (SchulteMarkus) [reply docs/publishing-your-site.md:94]: Well I just had my first time setup and had to do this setting. I can replay it one a fresh repository, but I assume you want to check yourself before (if) applying this change.

Comment (squidfunk) [reply docs/publishing-your-site.md:94]: I see. Could we maybe reformulate the instructions saying "if the page doesn't show up after a few minutes, check your `gh-pages` setting here...", so maybe more like troubleshooting advice.

**Review by squidfunk — APPROVED** [review]

**Comment by squidfunk** [comment]
Thanks!

**Pull request merged**